tests/plugins/emoji/manual/dropdown/pointercursorinnavigation.md

```diff
@@ -1,21 +1,12 @@
-@bender-tags: 4.11.2, bug, emoji, 2592
+@bender-tags: 4.11.3, bug, emoji, 2592
 @bender-ckeditor-plugins: wysiwygarea, toolbar, elementspath, sourcearea, emoji, clipboard, undo, stylescombo, format
 @bender-ui: collapsed
 @bender-include: ../../_helpers/tools.js
 
-## Test Case 1:
-
 1. Open emoji dropdown.
 2. Move cursor over group navigation at top of dropdown.
 ### Expected:
 Cursor is changed to pointer, which indicates that element is a link.
 3. Move cursor over emoji in main section.
 ### Expected:
 Cursor is changed to pointer, which indicates that element is a link.
-
-## Test Case 2:
-
-1. Open emoji dropdown.
-2. Click into navigation group.
-### Expected:
-Emoji list is scrolled to proper position. First element of clicked group is selected.
```

Comment:
Commit message says it should be splitted in two, but I see only one test.
Am I missing something, or this part is just removed?

Reply:
I've add tag `4.11.3` to old test which covers this behaviour. (`tests/plugins/emoji/manual/dropdown/scrollpagebug.md`). I feel that is better to reuse already existing test, rather than write new one.
However I see that test had invalid description so I add more precise info to expected and unexpected value.